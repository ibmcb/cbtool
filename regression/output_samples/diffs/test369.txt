Current experiment identifier is "EXP-08-21-2013-04-12-27-PM-EDT".
