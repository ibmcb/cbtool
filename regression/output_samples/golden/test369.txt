Current experiment identifier is "EXP-08-21-2013-02-53-55-PM-EDT".
